gradle/findbugs-exclude.xml

```diff
@@ -50,6 +50,7 @@
       <Bug pattern="EC_NULL_ARG"/>
       <Bug pattern="NP_ALWAYS_NULL"/>
       <Bug pattern="MS_CANNOT_BE_FINAL"/>
+      <Bug pattern="MS_PKGPROTECT"/>
```

Comment:
I am curious why we add `MS_PKGPROTECT`?

Reply:
This came out of migrating to scala 2.12 (a requirement when migrating to kafka 2.5)
Spotbugs checks fails on scala compiled classes in this new version.
https://github.com/gradle/gradle/issues/8308 seems to report a similar issue noticed kafka itself excludes this check also in their spotbugs configuration.